### ISSUE TITLE
Question about Hard Sector - NorthStar Image

### ISSUE THREAD

**Issue by ldkraemer** [davidgiven/fluxengine#283]
I have several Northstar images in .NSI format (Sector Dumps).  These floppy's are 10 Sectors (Hard Sectors)
and are 512 Bytes per sector for 35 Sectors times 2 Sides - (((10 * 512) * 35) * 2) =  358,400 Bytes per
Image.

If I use HxC to load the RAW Sector Dump and create a valid *.SCP file will it be possible to write a floppy
with the Fluxengine that a Northstar will accept?

Thanks.

Larry

**Comment by davidgiven** [comment]
I have no idea about HxC --- sorry. FluxEngine can natively handle .nsi images so you should be able to write them directly. For your files, use the northstar350 profile. Also, please let me know if it works, as I'm not sure this has been tested in the wild.

Regarding writing SCP files: yes, this can work, but you have to prepare them correctly (they must contain exactly one revolution of data). Also, this is even less tested!

**Comment by hharte** [comment]
Writing .nsi files to disks should work.  Please make sure to use a 300RPM 48TPI drive, and adjust the spindle speed to exactly 300RPM.

Also make sure the FluxEngine hardware is programmed with the latest firmware, there was a recent fix to the hard sector pulse counting.

-Howard

**Comment by jfdelnero** [comment]
@ldkraemer  Yes the HxC software support Hard-sectored NSI images, but the SCP files with the current format specification can't support Hard sectored disk (No sector index information). Use the HFEv3, Kryoflux Raw flux or the HxC Pauline dumper stream file format instead.
